utilities/ssp.py

```diff
@@ -299,3 +303,15 @@ def verify_ssp_pod_is_running(
         else:
             LOGGER.error(f"SSP pod was not running for last {TIMEOUT_6MIN} seconds")
             raise
+
+
+def cluster_instance_type_for_hot_plug(guest_sockets: int, cpu_model: Optional[str]):
```

Comment:
as for `cpu_model`:
1. please use python's recommended syntax of `x|y` ( as of 3.10)
2. missing default value

Reply:
Don't think we need a default value here